tests/requirements-tools.txt

```diff
@@ -1,11 +1,16 @@
 # Work-around for a bug in execnet 1.4.1
 git+https://github.com/pytest-dev/execnet.git
 
+# TMP require newer pytest for win32 3.6
+# See pytest-dev/pytest#103
+git+https://github.com/pytest-dev/pytest.git ; sys_platform == 'win32' and python_version >= '3.6'
```

Comment:
A newer pytest is still required though due to another unrelated issue (05796be21a4421e5f76893afd34140706a71b42b).